mypy/typeshed/stdlib/_typeshed/__init__.pyi

```diff
@@ -184,3 +184,7 @@ else:
     @final
     class NoneType:
         def __bool__(self) -> Literal[False]: ...
+
+# Actually the type of Ellipsis is <type 'ellipsis'>, but since it's
```

Comment:
This file is just synced from here: https://github.com/python/typeshed/blob/master/stdlib/_typeshed/__init__.pyi

Manually updating it is not a good practice.